Backend/app/models/partner.model.js

```diff
@@ -1,38 +1,54 @@
-const sql = require('./db.js');
-const logger = require('../logger');
+const sql = require("./db.js");
+const logger = require("../logger");
 
-// constructor for partners 
+// constructor for partners
 const Partner = function (body) {
-    this.name = body.name;
-    this.city = body.city;
-    this.village = body.village;
-    this.address = body.address;
-    this.phone = body.phone;
+  this.name = body.name;
+  this.city = body.city;
+  this.village = body.village;
+  this.address = body.address;
+  this.phone = body.phone;
 };
 
 // add new partner
-Partner.prototype.create = function() {
-    return new Promise((resolve, reject) => {
-        sql.query('INSERT INTO partners (org_name, city, village, address, phone) VALUES (?)',
-        [[this.name, this.city, this.village, this.address, this.phone]],
-        function(err, results) {
-            if (err) reject (err);
-            else resolve(results.insertId);
-        });
-    });
-}
+Partner.prototype.create = function () {
+  return new Promise((resolve, reject) => {
+    sql.query(
+      "INSERT INTO partners (org_name, city, village, address, phone) VALUES (?)",
+      [[this.name, this.city, this.village, this.address, this.phone]],
+      function (err, results) {
+        if (err) reject(err);
+        else resolve(results.insertId);
+      }
+    );
+  });
+};
```

Comment:
Try not to use a two-dimensional array for this application, instead use multiple `?`s and a one-dimensional array.